rotkehlchen/externalapis/cryptocompare.py

```diff
@@ -220,6 +222,10 @@ def can_query_history(
             source=HistoricalPriceOracle.CRYPTOCOMPARE,
         )
         got_cached_data = data_range is not None and data_range[0] <= timestamp <= data_range[1]
+        is_penalized = self.is_penalized()
+        if is_penalized is True and got_cached_data is False:
```

Comment:
no need to create a new variable here
```suggestion
        if self.is_penalized() is True and got_cached_data is False:
```